mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_bom_total_views.xml

```diff
@@ -10,5 +11,48 @@
                 </xpath>
             </field>
         </record>
+
+        <record id="mrp_unbuild_bom_quants_reduced_view" model="ir.ui.view">
+            <field name="name">mrp.unbuild.bom.quants.view.tree (from mrp_unbuild_cust_alu_analytics)</field>
+            <field name="priority" eval="100"/>
+            <field name="model">mrp.unbuild.bom.quants</field>
```

Comment:
This view belongs to a model that doesn't match to XML file. Please place it in the right file.

Reply:
Still pending this

Reply:
Done